### PR TITLE
Pedantic cleanup.

### PR DESCRIPTION
**Pull request by shmookey**
Fix formatting, spelling errors etc.

### CODE REVIEW

**Comment by stephanh** [comment]
This file is pretty out of date.

File: doc/code.md [hunk 1]
```diff
@@ -1,59 +1,64 @@
...
+   external API.
+ - `example` contains end-to-end usage examples of maestro. New functionality
+   should be added to the examples, stubbed out and filled in top-down from
+   `core/macros/api`.
+ - `benchmark` is useful for micro-benchmarks, this has been important for 
+   working on codecs in particular.
 
 
-## development lifecycle
+## Development lifecycle
```
Comment (stephanh) [inline doc/code.md]: Lets delete this whole section.

File: doc/code.md [hunk 2]
```diff
@@ -1,59 +1,64 @@
...
- - Partition.byFields* needs to be cleaned up. The goal is sane error messages, at the moment
-   that is only achievable by the non-overloaded, duplicated versions. It might be worth def macro
-   a single version and writing a custom error message.
+## TODOs
 
+ - Partition.byFields* needs to be cleaned up. The goal is sane error messages,
+   at the moment that is only achievable by the non-overloaded, duplicated
+   versions. It might be worth def macro a single version and writing a custom
+   error message.
  - Clean-up TemplateParquetSource and move to ebenezer.
```
Comment (stephanh) [inline doc/code.md]: Done, delete.

File: doc/code.md [hunk 3]
```diff
@@ -1,59 +1,64 @@
...
-
  - Add tests around error handling.
+ - Autoify the macro codecs via MacroSupport, requires finding work around for
+   macro-paradise bug on 2.10.3
+ - Revisit DecodeMacro, and tidy up by trying to pull out for comprehension
+   pattern
+ - Fix perf of Fields, by using vampire methods trick to get rid of structural
+   type
+ - Add a CascadeJob base class to work around scalding bug w.r.t. validating
```
Comment (stephanh) [inline doc/code.md]: Not using it anymore, please delete.

**Comment by stephanh** [comment]
:+1:.

File: doc/code.md (hunk 1 above)
Comment (shmookey) [inline doc/code.md]: Removed.

File: doc/code.md (hunk 2 above)
Comment (shmookey) [inline doc/code.md]: Removed.

File: doc/code.md (hunk 3 above)
Comment (shmookey) [inline doc/code.md]: Removed.

**Comment by stephanh** [comment]
This PR is a bit broken. Sam's commits are already merged in. @shmookey can we fix it up and then merge it.

**Comment by stephanh** [comment]
@shmookey can we do something about this PR please. It has grown a bit stale.

**Comment by shmookey** [comment]
Withdrawing since the information in this file is so out of date, and this PR only fixes formatting.

**Pull request closed without merging**